combat-trainer.lic

```diff
@@ -297,7 +299,8 @@ class SetupProcess
   end
 
   def determine_whirlwind_weapon(game_state)
-    offhand_skill = game_state.sort_by_rate_then_rank(game_state.whirlwind_trainables - [game_state.weapon_skill]).first
+    return if game_state.weapon_skill.include?'Twohanded'
```

Comment:
What do you think about making an array for 'two handers' (similar to line 3217) and using it here and the next line?